lms/templates/main.html

```diff
@@ -116,16 +98,24 @@
   <meta name="path_prefix" content="${EDX_ROOT_URL}">
   <meta name="google-site-verification" content="_mipQ4AtZQDNmbtOkwehQDOgCxUUV2fb_C0b6wbiRHY" />
 
-  <%include file="${google_analytics_file}" />
-
-% if style_overrides_file:
-  <link rel="stylesheet" type="text/css" href="${static.url(style_overrides_file)}" />
+<% ga_acct = microsite.get_value("GOOGLE_ANALYTICS_ACCOUNT", settings.GOOGLE_ANALYTICS_ACCOUNT) %>
```

Comment:
If a microsite doesn't want this Google Analytics code to be inserted, they can simply set `GOOGLE_ANALYTICS_ACCOUNT = None` in their microsite configuration. Then, they can use any analytics system they want, including Segment, simply by inserting *that* Javascript in the templates that the microsite uses.

I agree that this logic should be removed from the main template in the future. However, we're trying to maintain backwards compatibility as much as reasonable, and this is an easy thing to keep around. In the future, as comprehensive theming gains more and more features, it will be easier to start dismantling the microsites framework and migrating existing microsites to use comprehensive theming, instead.